### PR TITLE
Update elasticsuite-tracking-indices-causes-problems-with-elasticsear…

### PR DESCRIPTION
**Pull request by romainruaud**
…ch.md

## Purpose of this pull request
 
Replace PR #140 because target page has moved.
 
## Affected Support KB pages
 

 
https://support.magento.com/hc/en-us/articles/360034921492

### CODE REVIEW

**Comment by almarchenko** [comment]
@romainruaud  thank you! I'm sorry that we moved it before merging your PR, will process this

**Comment by almarchenko** [comment]
Requested review from Adobe Commerce SMEs.

File: src/troubleshooting/elasticsearch/elasticsuite-tracking-indices-causes-problems-with-elasticsearch.md [hunk 1]
```diff
@@ -29,6 +31,26 @@ ElasticSuite has a new feature that creates tracking indices. These tracking ind
...
+
+The default retention period is 365 days. You can reduce it to 30 or 15 days.
+
+### Upgrade your ElasticSuite version to use monthly based indices instead of daily based indices
+
+Once you have upgraded the ElasticSuite plugin to version > 2.9.8 / 2.10.7, tracking indices will be monthly based.
+
+You can still reduce the retention period : 
+
+Go to **Stores > Configuration > Tracking > Global Configuration > Retention Delay**
```
Comment (almarchenko) [inline src/troubleshooting/elasticsearch/elasticsuite-tracking-indices-causes-problems-with-elasticsearch.md:48]:
```suggestion
Go to **Stores** > **Configuration** > **Tracking** > **Global Configuration** > **Retention Delay**
```

File: src/troubleshooting/elasticsearch/elasticsuite-tracking-indices-causes-problems-with-elasticsearch.md [hunk 2]
```diff
@@ -29,6 +31,26 @@ ElasticSuite has a new feature that creates tracking indices. These tracking ind
 
 ## Solution
 
+### Upgrade your ElasticSuite version to be able to cleanup tracking indices periodically
+
+Once you have upgraded the ElasticSuite plugin to version > 2.8.0, you can configure a periodical cleaning of indices. 
+
+Go to **Stores > Configuration > Tracking > Global Configuration > Retention Delay**
```
Comment (almarchenko) [inline src/troubleshooting/elasticsearch/elasticsuite-tracking-indices-causes-problems-with-elasticsearch.md:38]:
```suggestion
Go to **Stores** > **Configuration** > **Tracking** > **Global Configuration** > **Retention Delay**
```

**Review by almarchenko — CHANGES_REQUESTED** [review]
Please commit the suggested changes and I would be able to import and merge.
Thank you

**Comment by romainruaud** [comment]
Hi, I updated the requests

**Comment by almarchenko** [comment]
@magento import code to magento-kb/knowledge-base

**Pull request merged**